server/routes/apiAuthRoutes.js

```diff
@@ -52,6 +52,23 @@ router.use( function ( req, res, next ) {
                 // user is authorized!
                 req.user = user;
 
+                // Hook the send function so that we can append the user to the body
+                res.hooks.on( 'send', ( args ) => {
+                    let responseJson = JSON.parse( args );
+
+                    // authenticated user object - add anything you need here
+                    let authUser = { 
+                        userId: user['userId'], 
+                        firstName: user['firstName'], 
+                        lastName: user['lastName'], 
+                        email: user['email'] 
+                    };
+
+                    responseJson['authUser'] = authUser;
+
```

Comment:
I have been playing with this for a bit using the http://localhost:4200/api/v1/auth/workspaces call as a test. I am not getting back the authUser information. I get into this block, but do not see anything in the response.